graphdatascience/graph/graph_proc_runner.py

```diff
@@ -400,7 +400,7 @@ def construct(
         nodes: Union[DataFrame, List[DataFrame]],
         relationships: Union[DataFrame, List[DataFrame]],
         concurrency: int = 4,
-        undirected_relationship_types: Optional[List[str]] = None,
+        undirected_relationship_types: List[str] = [],
```

Comment:
Personal opinion: Make be better to keep the optional since the list is not recreated every method call

Reply:
OK done